gemfiles/rails5.1.gemfile

```diff
@@ -2,4 +2,3 @@
 eval_gemfile('./common.rb')
 
 gem 'activerecord', '~> 5.1.0'
-gem 'mysql2', '>= 0.3.18', '< 0.6.0', platforms: :ruby
```

Comment:
These mysql2 version restrictions are actually taken from activerecord’s Mysql2Adapter files, e.g. https://github.com/rails/rails/blob/5-1-stable/activerecord/lib/active_record/connection_adapters/mysql2_adapter.rb#L4, because while Rails cannot have an explicit requirement on mysql2 (it doesn’t know which database or adapter you’ll use), ActiveRecord actually _does_ apply a version restriction at load time.

Reply:
Understood. I went with a simpler specifier mostly to match our actual work & testing... unless we're gonna expand the matrix to a 3rd dimension to test against multiple versions of mysql? I don't forsee us doing that anytime soon tho.